### PR TITLE
Fix several of the asymptote errors

### PR DESCRIPTION
**Pull request by mmatera**
Here I also fixed several of the asymptote errors. In my machine, the test passed flawlessly, but in Travis it is still fails. I would appreciate it if someone else could check it independently.

### CODE REVIEW

**Comment by GarkGarcia** [comment]
> In my machine, the test passed flawlessly, but in Travis it is still fails.

This may be cause by the fact that you probably have a more feature-full LaTeX installation in your machine than `texlive-xetex` (such as `texlive-full`). I'll wait for the builds to finish to investigate more.

File: .travis.yml [hunk 1]
```diff
@@ -57,8 +57,10 @@ before_install:
     fi
   - if [[ "$DOC" == "true" ]]; then
       sudo apt-get install -qq asymptote texlive-latex-extra texlive-latex3 texlive-fonts-recommended;
-      sudo apt-get install -qq  texlive-font-utils latexmk texlive-xetex lmodern;
+      sudo apt-get install -qq texlive-font-utils latexmk texlive-xetex lmodern;
+      sudo apt-get install -qq graphviz;
       sudo add-apt-repository -y ppa:inkscape.dev/stable-daily && sudo apt-get update && sudo apt-get install -qq inkscape;
+      pip install pydot;
```
Comment (GarkGarcia) [inline .travis.yml:63]: I think this line is unnecessary.

Comment (mmatera) [reply .travis.yml:63]: I have not added it, but OK, lets remove it...

Comment (GarkGarcia) [reply .travis.yml:63]: Yes, it was me, my bad 😅️

**Comment by mmatera** [comment]
I made a rebase to the master, and now it raise an error, comming from the graph package. In anycase, the problem in the compilation of the latex/asympotote file seems to be related to the asymptote version, and is restricted to 3D graphics. With a newer version it compiles...

**Comment by GarkGarcia** [comment]
> I made a rebase to the master, and now it raise an error, comming from the graph package. In anycase, the problem in the compilation of the latex/asympotote file seems to be related to the asymptote version, and is restricted to 3D graphics. With a newer version it compiles...

Ok, should I merge it then?

**Comment by rocky** [comment]
>  With a newer version it compiles...

What does *it* refer to? A newer version of asymptote? What version is good and what version isn't?

**Comment by mmatera** [comment]
> > With a newer version it compiles...
> 
> What does _it_ refer to? A newer version of asymptote? What version is good and what version isn't?

I am sorry: in my system I have  asymptote v2.41-4_amd64  and it compiles all the examples. 
In travis, the version that is installed with apt is  2.37-1_amd64   and fails to compile the examples involving 3d objects.

**Comment by rocky** [comment]
Well, we probably could switch TravisCI from Xenial to Bionic or Focal https://docs.travis-ci.com/user/reference/focal/ and that probably would be a good thing and a better fit for Python 3.6+ and good in the long run.

**Comment by GarkGarcia** [comment]
> Well, we probably could switch TravisCI from Xenial to Bionic or Focal (now that 2.7 has been dropped) https://docs.travis-ci.com/user/reference/focal/ or https://docs.travis-ci.com/user/reference/focal/#python-support and that probably would be a good thing and a better fit for Python 3.6+ and good in the long run.

I agree. Let's see if https://github.com/mathics/Mathics/pull/878 helps with this.

**Comment by GarkGarcia** [comment]
[This build](https://travis-ci.org/github/mathics/Mathics/jobs/726766214#L2491) failed because a connection was reset when downloading a dependency. This isn't related to the changes in the PR.

**Comment by GarkGarcia** [comment]
[This](https://travis-ci.org/github/mathics/Mathics/jobs/726766219#L445) also failed because of an issue regarding Inkscape's PPA. This will be fixed by https://github.com/mathics/Mathics/pull/880.

**Comment by GarkGarcia** [comment]
https://github.com/mathics/Mathics/pull/880#issuecomment-691683393 is the only thing left for us to address. I'll merge this PR and we can fix those errors in other threads.

**Pull request merged**